### DISCUSSION TITLE
Notifications, Filtering, and GroupBy

### DISCUSSION THREAD

**Discussion by BenjamenMeyer** [community/community#12977] [Other Feature Feedback, Questions, & Ideas]
I love the the notifications screen, and when in the default view (Inbox) I use the `GroupBy:repositories` feature. However, I also want to use the filters so I can look by Organization as I'm involved in several organizations and want to be able to concentrate on those at times; however, this disables the `GroupBy` feature so I just get a long list of things to look at. It'd be awesome if the GroupBy was able to work even with the filters.

BTW, my general workflow:
- go to notifications
- look over the repositories that have notifications
- click 'Done' for all items on the repositories I care least about but still want to know about changes to
- review the repositories I care most about

I've recently started using the `org` filter, which really helps limit things for the various orgs I'm part of and reduce distractions. However, the orgs each have multiple repositories and I'm still more interested in some repositories than others, and the `GroupBy:repository` was extremely helpful in being able to prioritize which PRs, Issues, etc I look at based on what I'm specifically tasked with working on. It'd be great to be able to have these features work together.

NOTE: I talk about using the `org` filter, but this could really apply to any filter in use.

**Comment by justinmchase** [comment]
I am basically doing something very similar. I have so many notifications its very hard to triage them effectively because there is so much noise and the filtering / grouping capabilities are limited.

In addition to to the grouping mentioned by @BenjamenMeyer I am really wishing there was a way to filter out pull requests in the `merged` state. Meaning, when I look at my notifications I'd really like to only like to see the ones that are still open. Merged PR notifications is fine, especially when they're my PR's but I just really need to see all the ones that are currently open the most. I literally have hundreds of PR's I need to triage and review per day and don't have time to sort through merged PRs.

**Comment by ihnorton** [comment]
Related discussions:
- https://github.com/refined-github/refined-github/issues/3781
- https://github.com/community/community/discussions/15591

**Comment by bsipocz** [comment]
I was about to ask for this same, please add "group-by org" as an option on top of the current "group by repo" and "group by date". I know I can use filters, but that's very limited compared to a global view of all notifications.
